G/GDAL/common.jl

```diff
@@ -0,0 +1,142 @@
+using BinaryBuilder, Pkg
+
+function configure(version_offset, min_julia_version, proj_jll_version)
+    # The version of this JLL is decoupled from the upstream version.
+    # Whenever we package a new upstream release, we initially map its
+    # version X.Y.Z to X00.Y00.Z00 (i.e., multiply each component by 100).
+    # So for example version 2.6.3 would become 200.600.300.
+
+    name = "GDAL"
+    upstream_version = v"3.2.1"
+    version = VersionNumber(upstream_version.major * 100 + version_offset.major,
+                            upstream_version.minor * 100 + version_offset.minor,
+                            upstream_version.patch * 100 + version_offset.patch)
+
+    # Collection of sources required to build GDAL
+    sources = [
+        ArchiveSource("https://github.com/OSGeo/gdal/releases/download/v$upstream_version/gdal-$upstream_version.tar.gz",
+            "43d40ba940e3927e38f9e98062ff62f9fa993ceade82f26f16fab7e73edb572e"),
+        DirectorySource("./bundled"),
```

Comment:
Ok, I can confirm using `../bunlded` is innocuous:
```suggestion
        DirectorySource("../bundled"),
```